packer/scripts/lab_setup.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+set -e
+set -x
+
+WORKSHOP=streaming-data-workshop
+WORKSHOP_DIR=${HOME}/${WORKSHOP}
+
+git clone https://github.com/infinispan-demos/${WORKSHOP} ${WORKSHOP_DIR}
+
+cd ${WORKSHOP_DIR}
+mvn install dependency:go-offline
+
+cd ${WORKSHOP_DIR}/web-viewer
+nvm use 4.2
+npm install
+npm run build
+
+cd ${WORKSHOP_DIR}
+./start-openshift.sh
+./start-datagrid.sh
```

Comment:
I'm not sure about the role of this `lab_setup`, but `start-datagrid.sh` should not be executed by attendees. This is mostly there to start the datagrid via script, but attendees should do that via UI.